### ISSUE TITLE
Watch screen is empty after loading image and JSON in editor

### ISSUE THREAD

**Issue by ovik88** [chm-dev/GTS-watchface-bundle#1]
Hey buddy,
nice tutorial but im facing some issus... Once i run editor and try to upload image and json from unpacked .bin file theres nothing loaded at all... have you came to same issues? any idea what could be wrong? thank you :-)

**Comment by nokhzz** [comment]
Image for prorety not found? I have this error too

**Comment by ovik88** [comment]
No, there is no error... Just buttons are green as everything is fine but there is no preview and seems like nothing is loaded
